### ISSUE TITLE
Failed at running `yarn preprocess`

### ISSUE THREAD

**Issue by sdelvalle57** [eth-infinitism/bundler#184]
Im getting this issue as trying to run `yarn preprocess` I believe the issue is when calling `yarn submodule-update` 

this is the error log:

```
yarn run v1.22.19
$ git submodule update --remote --init --recursive
fatal: Needed a single revision
fatal: Unable to find current origin/fix-deployment revision in submodule path 'submodules/account-abstraction'
error Command failed with exit code 1.
```

**Comment by dbrans** [comment]
+1

**Comment by drortirosh** [comment]
fixed by #185
